_data/simple-icons.json

```diff
@@ -271,6 +271,10 @@
             "source": "https://circleci.com/press"
         },
         {
+            "title": "Clockify",
+            "hex": "03A9F4",
+            "source": "https://clockify.me/"
+        },        {
```

Comment:
Could you make sure that the formatting matches the rest of the file? The closing curly bracket should have a newline after the comma.